### ISSUE TITLE
Hint about gopls multi-module workspace support

### ISSUE THREAD

**Issue by ottob** [sublimelsp/LSP#1471]
**Describe the bug**
Not really a bug but a hint for other go users.

**To Reproduce**
1. have a single git repo with many go modules.
2. try to use lsp features like goto definition.
3. fail with errors like `no object found for ident X`

**Expected behavior**
LSP features to work.

**Additional context**
In gopls [v0.5.3](https://github.com/golang/tools/releases/tag/gopls%2Fv0.5.3) an experimental mode for multi modules was added. I enabled it in my sublime lsp config like this:

```json
"gopls":
	{
		"enabled": true,
		"initializationOptions": {
			"experimentalWorkspaceModule": true
		}
	},
```
Then goto definition etc works as expected.

**Comment by rwols** [comment]
If you are feeling adventurous, consider updating docs/index.md in this repository with your knowledge, and make a pull request.

**Comment by rwols** [comment]
Fixed by #1490
